bin/build-manylinux.sh

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env bash
+
+# This script builds BlazingMQ and all of its dependencies.
+#
+# Before running this script, install following prerequisites, if not present
+# yet, by copy-and-pasting the commands between `<<PREREQUISITES` and
+# `PREREQUISITES` below:
+<<PREREQUISITES
+sudo yum install -y \
+    gdb \
+    curl \
+    pkgconfig \
+    ninja-build \
+    flex-devel \
+    zlib-devel \
+    m4
+PREREQUISITES
+                                                    # shellcheck disable=SC2188
+
+set -e
+set -u
+[ -z "$BASH" ] || shopt -s expand_aliases
+
+# :: Set some initial constants :::::::::::::::::::::::::::::::::::::::::::::::
+DIR_ROOT="${DIR_ROOT:-$(pwd)}"
```

Comment:
Along the lines of [commit `179be2e`](https://github.com/bloomberg/blazingmq/commit/179be2ee766f07c6b0d2ca28f40fb7c0e11e921c), we probably don't want this to be user-configurable.